.github/workflows/llms-ctx.yml

```
@@ -0,0 +1,79 @@
+
+  build-ctx:
+    needs: validate
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+  schedule:
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 53: did not find expected key

(syntax-check)